torch/nn/parallel/distributed.py

```diff
@@ -432,9 +432,10 @@ def _sync_params(self):
 
             # module buffer sync
             if self.broadcast_buffers and len(self.modules_buffers[0]) > 0:
-                # cross-node buffer sync
-                self._dist_broadcast_coalesced(self.modules_buffers[0],
-                                               self.broadcast_bucket_size)
+                # Synchronize buffers across processes.
+                # The process with rank 0 is considered the authoritative copy.
+                self._broadcast_coalesced(self.modules_buffers[0],
```

Comment:
This is where it might be confusing. We have two `broadcast_coalesced`, but one is distributed and another is local. Is this intentional?

Reply:
They come from different modules. For clarity we can proxy them with a function on the class, one for local broadcast and the other for distributed broadcast.